lib/topological_inventory/ansible_tower/receptor/api_object.rb

```diff
@@ -82,7 +82,6 @@ def endpoint
 
       def build_payload(http_method, path, params = nil, receptor_opts = {})
         slug = path.to_s
-        slug += "?#{params.to_query}" if params
```

Comment:
@mkanoor why? How can you send the "id__in=" and other filters?

Reply:
or page_size?

Reply:
@slemrmartin They all go into params. All the Query Values go into params. By adding it on the url line you are adding extra work which is not needed.